### DISCUSSION TITLE
want to switch the github page url from domain_name.me/repo_name to user_name.github.io/repo_name( i.e. default url of the github pages )

### DISCUSSION THREAD

**Discussion by DevWithAnurag** [community/community#82809] [Other Feature Feedback, Questions, & Ideas]
### For resting to default URL of GitHub pages

General

### My Issue

I am facing issues with the GitHub URL. The problem begins when I claim the student developer pack free domain provided by NameCheap. As soon as I claim it, the domain name is attached to my GitHub Pages URL. For example, before my GitHub Pages URL was 'user_name.github.io/repo_name,' and after claiming, my GitHub Pages URL becomes 'domain_name.me/repo_name.'

I just want to change the URL back to the default.

here is image of the issue i am facing

removing the custom domain doesn't work

![Screenshot (71)_LI](https://github.com/community/community/assets/120032315/51399bbf-cdb9-4365-bd6b-7c84d7795e33)

**Comment by georgetoloraia** [comment]
To revert your GitHub Pages URL back to the default format („username.github.io/repository„), you should be able to remove the custom domain from your repository's GitHub Pages settings. However, you've indicated that simply removing the custom domain doesn't work. This can happen if the DNS settings for your custom domain are still pointing to GitHub's servers, or if there's a caching issue.

Here's a step-by-step approach to resolve this:

1. **Remove Custom Domain:**
- Go to the repository settings where your GitHub Pages site is published.
- Scroll down to the "GitHub Pages" section.
- If you see the custom domain listed, click the "Remove" button next to it.

2. **Check DNS Settings:**
- Go to your domain registrar's website (NameCheap in your case) and check the DNS settings.
- Make sure that there are no `CNAME `or `A` records pointing to GitHub's servers. If there are, remove them or reset the DNS settings to the default.

3. **Wait for DNS Propagation:**
- After making changes to DNS settings, it can take up to 48 hours for the changes to propagate across the internet. Until this propagation is complete, you may still see the custom domain being used.

4. **Clear Browser Cache:**
- Sometimes the issue can be due to your browser caching the old URL. Clear your browser cache to ensure it's not causing the issue.

5. **Check GitHub's Cache:**
- GitHub might cache the custom domain setting. If you've removed the custom domain but the change hasn't taken effect, wait for a short period, and then check again.

6. **Rebuild GitHub Pages:**
- Push a small change to your repository or use the "Rebuild Page" option in the repository settings to force GitHub to rebuild your page. This can sometimes help to reset the URL.

don't forgot bro MARK ANSWER ;)

**Reply by hamzashaukat111** [reply]
I have same issue, but I dont have any "Make sure that there are no CNAME or A records pointing to GitHub's servers. If there are, remove them or reset the DNS settings to the default." in my namecheap domain, still cant change back to orignal github pages it is not working.

**Reply by DevWithAnurag** [reply]
just change the user name of user GitHub it's  work for me

**Reply by hamzashaukat111** [reply]
Well yess, the name changing works. but if I change back to that name the problem still exists.
That name was all over the profile so kind of neededd.
Can anyone tell me about this, because I think this commit is the reason each of my new deployment to github pages is refering to that namcecheap domain I got, can anyone suggest me should I delete it or something, so that even without changing name it is back to orignal github pages.
<img width="684" alt="image" src="https://github.com/community/community/assets/75229817/187454af-c4a4-473b-9756-0616b4f14c2f">

**Comment by Salim255** [comment]
To revert your GitHub Pages URL back to the default format („username.github.io/repository„), you should be able to remove the custom domain from your repository's GitHub Pages settings. However, you've indicated that simply removing the custom domain doesn't work. This can happen if the DNS settings for your custom domain are still pointing to GitHub's servers, or if there's a caching issue.

**Reply by DevWithAnurag** [reply]
Yha ! i  try many of the tutorials  and the documentations ,Stack over Flow  but i am not able to solve the problem . so  after so many try i changed the name
